frameworks/kafka/src/main/resources/kafka_svc.yml

```diff
@@ -0,0 +1 @@
+../dist/kafka_svc.yml
```

Comment:
Oh, this isn't necessary.  You can rename back everything to svc.yml.  When I was trying to fix tests, things were picking up the wrong resource objects because they had the same name and I was having trouble figuring out what was going on.  That isn't a problem now.